package Protos;

message SRMNeuronC {
    required double u_rest = 1;
    required double amp_refr = 2;
    required double amp_adapt = 3;
    required double tau_refr = 4;
    required double tau_adapt = 5;
}

message SRMNeuronState {
    required double M = 1;
    required double gr = 2;
    required double ga = 3;
}

